spyderlib/config/main.py

```diff
@@ -586,9 +586,11 @@ def is_gtk_desktop():
               }),
             ('color_schemes',
              {
-              'names': ['Emacs', 'IDLE', 'Monokai', 'Pydev', 'Scintilla',
-                        'Spyder', 'Spyder/Dark', 'Zenburn'],
+              'names': ['emacs', 'idle', 'monokai', 'pydev', 'scintilla',

```

Comment:
You need to do a minor update in version in `CONF_VERSION` below (from `23.0.0` to `23.1.0`) for this change to take effect.